### PR TITLE
Merge file & filepath modules

### PR DESCRIPTION
**Pull request by ThishaniLucas**
$subject
Related issue: https://github.com/ballerina-platform/ballerina-standard-library/issues/480

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -28,6 +28,12 @@ ext.puppycrawlCheckstyleVersion = "8.18"
 ext.slf4jVersion = "1.7.30"
 ext.stdlibStringUtilsVersion = project.stdlibStringUtilsVersion
 ext.stdlibRuntimeVersion = project.stdlibRuntimeVersion
+ext.testngVersion = "6.14.3"
+ext.stdlibStringUtilsVersion = project.stdlibStringUtilsVersion
+ext.stdlibIOVersion = project.stdlibIOVersion
+ext.stdlibLogVersion = project.stdlibLogVersion
+ext.stdlibSystemVersion = project.stdlibSystemVersion
+ext.stdlibFileVersion = project.stdlibFileVersion
```
Comment (niveathika) [inline build.gradle:36]: Any reason to have file dependency?

File: build.gradle [hunk 2]
```diff
@@ -95,6 +125,10 @@ subprojects {
         /* Standard libraries */
         ballerinaStdLibs "org.ballerinalang:stringutils-ballerina:${stdlibStringUtilsVersion}"
         ballerinaStdLibs "org.ballerinalang:runtime-ballerina:${stdlibRuntimeVersion}"
+        ballerinaStdLibs "org.ballerinalang:io-ballerina:${stdlibIOVersion}"
+        ballerinaStdLibs "org.ballerinalang:log-ballerina:${stdlibLogVersion}"
+        ballerinaStdLibs "org.ballerinalang:system-ballerina:${stdlibSystemVersion}"
+        ballerinaStdLibs "org.ballerinalang:file-ballerina:${stdlibFileVersion}"
```
Comment (niveathika) [inline build.gradle:131]: Same

File: gradle.properties [hunk 3]
```diff
@@ -5,4 +5,8 @@ ballerinaLangVersion=2.0.0-Preview6-SNAPSHOT
 fileTransportVersion=6.0.55
 stdlibStringUtilsVersion=0.5.2-SNAPSHOT
 stdlibRuntimeVersion=0.5.2-SNAPSHOT
+stdlibIOVersion=0.5.2-SNAPSHOT
+stdlibLogVersion=1.0.2-SNAPSHOT
+stdlibSystemVersion=0.6.2-SNAPSHOT
+stdlibFileVersion=0.5.3-SNAPSHOT
```
Comment (niveathika) [inline gradle.properties:11]: Same

**Review by niveathika — APPROVED** [review]

**Comment by niveathika** [comment]
@ThishaniLucas Let's coordinate with dependant module owners and merge this. Update the dependency graph too

File: file-ballerina/src/file/filepath/windows_file_path.bal [hunk 4]
```diff
@@ -0,0 +1,290 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (kalaiyarasiganeshalingam) [inline file-ballerina/src/file/filepath/windows_file_path.bal:1]:
```suggestion
// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Check all other places as well

Comment (niveathika) [reply file-ballerina/src/file/filepath/windows_file_path.bal:1]: We are just moving the files, not committing newly. We can keep the license year as it is.

**Comment by ThishaniLucas** [comment]
Closing this PR and opening a new one with many other changes.

**Pull request closed without merging**